.github/workflows/deploy-tna-python.yml

```diff
@@ -0,0 +1,31 @@
+name: Deploy to AWS EC2 via SSM
+on:
+  push:
+    branches:
+      - main
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+    - name: Configure AWS credentials
+      uses: aws-actions/configure-aws-credentials@v3
+      with:
+        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-region: eu-west-2
+    - name: Pull and deploy the latest container image
+      run: |
+          aws ssm send-command \
+          --instance-ids i-0b9d358c28976acae \
+          --document-name "AWS-RunShellScript" \
+          --comment "Deploy latest container image" \
+          --parameters 'commands=[
+          "sudo docker pull ghcr.io/nationalarchives/tna-python:latest",
```

Comment:
This is an empty root image and not the application image. Is that right?

Reply:
We need to take these values from https://github.com/nationalarchives/ds-infrastructure-web/blob/main/config/develop.json#L3-L6

Reply:
Thanks for the update. Currently working on retrieving them. I will update once it's complete.